lib/graphql/lib/astToBody.js

```diff
@@ -10,21 +10,29 @@ export default function astToBody(ast) {
   return body;
 }
 
+function extractArguments(args) {
+  if (!Array.isArray(args) || args.length === 0) {
+    return {};
+  }
+
+  return args.reduce((acc, field) => {
+    acc[field.name.value] =
+      field.value.kind === 'ObjectValue'
```

Comment:
Can you please add a descriptive comment for this?

Reply:
done. Sorry for the delay.

Reply:
@theodorDiaconu, does the added comment work for you?